### ISSUE TITLE
positional arguments error 

### ISSUE THREAD

**Issue by fatimahdd** [bosskmk/pluto_grid#174]
the table was view to me and everything was great but now is not .
this message in debug console is shown :
Error: Too few positional arguments: 2 required, 1 given.
  `return painter.hitTestInteractive(localOffset);`
this is my code :
```dart
  List<PlutoColumn> columns = [
    PlutoColumn(
      title: 'project name ',
      field: 'project_name',
      type: PlutoColumnType.text(),
    ),
    PlutoColumn(
      title: 'UPN',
      field: 'UPN',
      type: PlutoColumnType.text(),
    ),
     PlutoColumn(
      title: 'status',
      field: 'status',
      type: PlutoColumnType.select([
        'active',
        'inactive',
      ]),
    ),
    PlutoColumn(
      title: 'created date',
      field: 'date_field',
      type: PlutoColumnType.date(),
    ),
  ];

  List<PlutoRow> rows = [
    
    PlutoRow(
      cells: {
        'project_name': PlutoCell(value: ' NUMS    '),
        'UPN': PlutoCell(value: '2021GP2470'),
        'status': PlutoCell(value: 'inactive'),
        'date_field': PlutoCell(value: '2019-08-06'),
      },
    ),
  ];
```

**Comment by jeansebastienZ** [comment]
Can you please take the time to edit your code correctly this way people will be able to read your problem and try to help you.

**Comment by bosskmk** [comment]
Could you please provide your `flutter --version` information so I can check the error?
```
C:\Users\manek\IdeaProjects\pluto_grid>flutter --version
Flutter 2.0.4 • channel stable • https://github.com/flutter/flutter.git
Framework • revision b1395592de (8 days ago) • 2021-04-01 14:25:01 -0700
Engine • revision 2dce47073a
Tools • Dart 2.12.2
```

And can you provide the entire stack of errors that occur?

**Comment by fatimahdd** [comment]
> Could you please provide your `flutter --version` information so I can check the error?
> 
> ```
> C:\Users\manek\IdeaProjects\pluto_grid>flutter --version
> Flutter 2.0.4 • channel stable • https://github.com/flutter/flutter.git
> Framework • revision b1395592de (8 days ago) • 2021-04-01 14:25:01 -0700
> Engine • revision 2dce47073a
> Tools • Dart 2.12.2
> ```
> 
> And can you provide the entire stack of errors that occur?



flutter --version
Flutter 2.1.0-13.0.pre.556 • channel master • https://github.com/flutter/flutter.git
Framework • revision 1a36c18497 (6 hours ago) • 2021-04-08 23:33:00 -0700
Engine • revision a9f4883dc8
Tools • Dart 2.13.0 (build 2.13.0-219.0.dev)
PS C:\Users\LENOVO\AndroidStudioProjects\projectCommittee>

**Comment by bosskmk** [comment]
You are using the master channel.
There was a change to the cupertinoScroll in the master channel and it caused an error.
PlutoGrid 2.0.0-pre.0 version with this modified will be released soon.
Maybe within a few hours.

**Comment by liqping** [comment]
Flutter 2.1.0-12.2.pre • channel beta • https://github.com/flutter/flutter.git
Framework • revision 5bedb7b1d5 (4 weeks ago) • 2021-03-17 17:06:30 -0700
Engine • revision 711ab3fda0
Tools • Dart 2.13.0 (build 2.13.0-116.0.dev)

./../.pub-cache/hosted/pub.flutter-io.cn/pluto_inside-1.0.0-pre.0/lib/src/widgets/pluto_scrollbar.dart:460:36: Error: Too many positional arguments: 1 allowed, but 2 found.
Try removing the extra positional arguments.
  return painter.hitTestInteractive(localOffset, kind) &&
                                   ^
../../.pub-cache/hosted/pub.flutter-io.cn/pluto_inside-1.0.0-pre.0/lib/src/widgets/pluto_scrollbar.dart:461:43: Error: Too many positional arguments: 1 allowed, but 2 found.
Try removing the extra positional arguments.
      !painter.hitTestOnlyThumbInteractive(localOffset, kind);
                                          ^

**Comment by bosskmk** [comment]
2.0.0-nullsafety.0 operates in the master channel (2.1.x).
Please use the 2.0.0-nullsafety.1 release now.

**Comment by NiranjanShah** [comment]
Flutter 2.2.0-10.3.pre • channel beta • https://github.com/flutter/flutter.git
Framework • revision 06e2fd6357 (6 days ago) • 2021-05-08 11:28:22 -0700
Engine • revision a123e75c60
Tools • Dart 2.13.0 (build 2.13.0-211.14.beta)


pluto_inside-0.2.0/lib/src/widgets/pluto_scrollbar.dart:460:36: Error: Too few positional arguments: 2 required, 1 given.
  return painter.hitTestInteractive(localOffset);
                                   ^

**Comment by NiranjanShah** [comment]
Can you please assist in resolving the above error.....

**Comment by bosskmk** [comment]
Please use Flutter's stable channel or fix the 2.0.0-nullsafety.1 version of plutoGrid.

plutoGrid is responding to errors in Flutter's stable channel.

**Comment by NiranjanShah** [comment]
Sorry didn’t understand the solution.

Do you want me to use stable channel with 2.0.0-nullsafety.1 version ?

**Comment by bosskmk** [comment]
`PlutoGrid : 2.0.0` on the stable channel
`PlutoGrid : 2.0.0-nullsafety.0` in beta or master channels where errors occur.

This error is caused by API changes in Flutter 2.1.x or later.

**Comment by sarbogast** [comment]
I'm using Flutter 2.2.0 in stable channel and I tried both 2.0.0 and 2.0.0-nullsafety.1 (and 2.0.0-nullsafety.0) version of PlutoGrid, and I still have that issue.

**Comment by bosskmk** [comment]
Resolved `PlutoGrid : 2.1.0`

**Comment by justkawal** [comment]
> I'm using Flutter 2.2.0 in stable channel and I tried both 2.0.0 and 2.0.0-nullsafety.1 (and 2.0.0-nullsafety.0) version of PlutoGrid, and I still have that issue.

Same here, I also used the version ``` plutoGrid 2.1.0``` with the other versions mentioned by @sarbogast 

I didn't worked for me
